<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body Mass Index Calculator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="gender_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="Gender_display">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="weight_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Weight (kg)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="weight_spinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="weight_Slider">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="height_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Height (cm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="height_spinBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="height_Slider">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="BMI_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Body Mass Index</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="BMI_display">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="interpretation_label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Interpretation</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="interpretation_display">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="action_Reset"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuGender">
    <property name="title">
     <string>Gender</string>
    </property>
    <addaction name="action_female"/>
    <addaction name="action_male"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuGender"/>
  </widget>
  <action name="action_Reset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_female">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </action>
  <action name="action_male">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>height_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>height_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weight_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>weight_Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
